invenio_drafts_resources/services/records/components/files.py

```diff
@@ -64,6 +67,14 @@ def update_draft(self, identity, data=None, record=None, errors=None):
                     ],
                 }
             )
+        elif draft.files.enabled and draft.files.items():
```

Comment:
If `items()` returns [] then the for loop below will not be entered. There is no need to collect items twice?
```suggestion
        elif draft.files.enabled:
```